src/ArrayObject.php

```diff
@@ -425,7 +425,7 @@ public function unserialize($data)
                     $this->setIteratorClass($v);
                     break;
                 case 'protectedProperties':
-                    continue;
+                    continue 2;
```

Comment:
Honestly, this is _more_ confusing, as it implies there are _two loops_, when, in this case case, it's actually a `switch` inside a `foreach`.

Let's make this a `break` instead, please, as that will achieve the same result: continuing on to the next item in the loop.

Reply:
updated